packages/tracing/src/browser/metrics/index.ts

```diff
@@ -38,6 +40,34 @@ export function startTrackingWebVitals(reportAllChanges: boolean = false): void
   }
 }
 
+/**
+ * Start tracking long tasks. Experimental.
+ */
+export function startTrackingLongTasks(reportLongTasks: boolean = false): void {
+  const transaction = getActiveTransaction() as IdleTransaction | undefined;
+  let po : PerformanceObserver | undefined = undefined;
+  const entryHandler: PerformanceEntryHandler = (entry: PerformanceEntry): void => {
+    if (transaction?.endTimestamp && po) {
```

Comment:
Checking for transaction being finished using end, if that still works, otherwise this just makes a span for long tasks, just keeping it simple to start, we can add attribution etc. later if it seems helpful.

Reply:
Why do we need to do this? Let's just attach it to any running transaction. Also we can't use optional chaining - it's increased final bundle as it's esnext feature.